src/main/java/liquibase/ext/databricks/sqlgenerator/CreateTableGeneratorDatabricks.java

```diff
@@ -49,7 +48,7 @@ public Sql[] generateSql(CreateTableStatement statement, Database database, SqlG
             if ((!StringUtils.isEmpty(thisStatement.getTableFormat()))) {
                 finalsql.append(" USING ").append(thisStatement.getTableFormat());
             } else if (thisStatement.getExtendedTableProperties() != null && StringUtils.isNoneEmpty(thisStatement.getExtendedTableProperties().getTblProperties())) {
-                finalsql.append(" TBLPROPERTIES (").append(thisStatement.getExtendedTableProperties().getTblProperties()).append(")");
+                finalsql.append(" TBLPROPERTIES (").append(thisStatement).append(")");
```

Comment:
this change need to be reverted, otherwise we gonna get `[package_path].CreateTableStatementDatabricks@object_hash` in the SQL statement